src/js/scatter-plot.js

```diff
@@ -17,16 +17,72 @@ const createRaceScatterPlot = () => {
     numTicks: 8,
     convert: num => num === 0 ? "0" : `${num / 1000}K`
   };
-  const toText = {
-    xHeader: "% Cash Bail",
-    yHeader: "Avg. Bail Amount",
-    x: value => `${value.toFixed(1)}%`,
-    y: value => value
+
+  const tooltipConfig = {
+    columns: [
+      { dataKey: "name", isHeader: true },
+      { header: "% Cash Bail", dataKey: "x", toText: value => `${value.toFixed(1)}%` },
+      { header: "Avg. Bail Amount", dataKey: "y", toText: value => value.toLocaleString("en", {
```

Comment:
The values are already formatted as we want them displayed. I'd rather we pass the raw value like before:
```js
x: x[key],
y: Number(y[key].replace(/[^\d.-]/g, "")),		
xText: this.toText.x(x[key]),
yText: this.toText.y(y[key]),
```

That way we don't have to worry about formatting messing up what we already know is correct.